kibana.md

```diff
@@ -542,6 +542,19 @@ readonlyrest_kbn.multiTenancyEnabled: false
 
 You can configure an ACL in multi tenancy mode by adding a few ACL blocks containing the `kibana.index` [rule](https://docs.readonlyrest.com/elasticsearch#kibana). See examples and further explanation under our [multi-tenancy guide](examples/multitenancy\_guide.md).
 
+### Extending the Kibana API with the x-ror-tenancy-id header
+([Enterprise](https://readonlyrest.com/enterprise))
+
+To target a specific tenant when making a [Kibana API](https://www.elastic.co/guide/en/kibana/current/api.html) request, include the custom HTTP header `x-ror-tenancy-id`. The value of this header should match one of the [groups rules](/elasticsearch#groups-rules) defined in your ACL configuration.
```

Comment:
We should also describe what happens when no group id is passed. 

And what happens when given group id is forbidden (because in the metadata response is not such group)

Reply:
```
What is the correct answer? The first group defined in the ACL for a specific user?
```
ah, right - the ROR ES determines it. Yes, this is the correct answer ;)

```
Isn't it redundant info?
```
yes, you are right

Reply:
> ah, right - the ROR ES determines it. Yes, this is the correct answer ;)

It's still worth mentioning here, right?

Reply:
yeah, let's add this info.

Reply:
Done ✔️